index.js

```diff
@@ -2,8 +2,11 @@
 var mapObj = require('map-obj');
 var camelCase = require('camelcase');
 
-module.exports = function (obj) {
+module.exports = function (obj, options) {
+	options = options || {};
+	options.exclude = options.exclude || [];
```

Comment:
`var exclude = options.exclude || []` and use the variable.

Right now you're modifying the user-supplied object, which is bad and will change the behavior of consecutive calls. Remember that objects are reference based.